scripts/update/Updater.php

```diff
@@ -18,12 +18,17 @@
  *
  */
 
-namespace oat\taoSync\taoSyncServer\update;
+namespace oat\taoSyncServer\update;
+
+use oat\taoSyncServer\install\RegisterServices;
 
 class Updater extends \common_ext_ExtensionUpdater
 {
     public function update($initialVersion)
     {
-
+        if ($this->isVersion('0.1.0')) {
+            $this->runExtensionScript(RegisterServices::class);
```

Comment:
we can't use an existing script. For the installer, it will be changed but for the updater, it should be static

Reply:
fixed